analyzer/windows/modules/auxiliary/officeregkeys.py

```diff
@@ -0,0 +1,145 @@
+# Copyright (C) 2016-2017 Cuckoo Foundation.
```

Comment:
what's the added value of running it all at once?

Reply:
Fix cases when Word -> Excel for example, and excel have macros

Reply:
ahh nice, document it somewhere